util/fqdn_test.go

```diff
@@ -31,7 +31,7 @@ func TestIsFQDNQuestionMarkFQDN(t *testing.T) {
 }
 
 func TestIsFQDNQuestionMarkIncorrectPlaceFQDN(t *testing.T) {
-	domain := "?.?.abc.?com"
+	domain := "?.?.abc?.com"
```

Comment:
I think govalidator stopped checking the suffix for things; I just pulled the update and a lot of changes were made. _sigh_